src/cloudformation/structure/cloudformation_parser.go

```diff
@@ -51,7 +52,11 @@ func (p *CloudformationParser) GetSupportedFileExtensions() []string {
 }
 
 func (p *CloudformationParser) ParseFile(filePath string) ([]structure.IBlock, error) {
-	template, err := goformation.Open(filePath)
+	template, err := goformation.OpenWithOptions(filePath, &intrinsics.ProcessorOptions{
+		StringifyPaths: []string{
+			"Resources/*/Properties/Environment/Variables/*",
```

Comment:
@rotemavni This can happen on other fields as well - this is a crucial const. Let's extract it to a const value somewhere